src/EndOfLifeDate/EndOfLifeDate.cs

```diff
@@ -5,8 +5,7 @@ namespace EndOfLifeDate;
 
 public class EndOfLifeDate
 {
-
-    private const string BaseUrl = "https://endoflife.date/api/";
+    private const string BaseUrl = "https://deploy-preview-5323--endoflife-date.netlify.app/api/";
```

Comment:
This is only for testing with `windows-server-core` and `windows-nano-server`. You might want to change this back for the future.